core/unix/memquery.h

```diff
@@ -104,15 +104,17 @@ memquery_iterator_next(memquery_iter_t *iter);
  */
 int
 memquery_library_bounds(const char *name, app_pc *start /*IN/OUT*/, app_pc *end /*OUT*/,
-                        char *fullpath /*OPTIONAL OUT*/, size_t path_size);
+                        char *fullpath /*OPTIONAL OUT*/, size_t path_size,
```

Comment:
Hmm, should we s/path/dir/ to make it clear it does not have the name?  So `fulldir` and `dir_size` (or `fulldir_size`), here and the other places.

Reply:
Ok